src/mysqlsh_helpers.py

```diff
@@ -38,3 +65,30 @@ def create_cluster(self):
     def add_instance_to_cluster(self):
         """Add an instance to the InnoDB cluster."""
         pass
+
+    def run_mysqlsh_script(self, script: str) -> AnyStr:
+        """Execute a MySQL shell script.
+
+        :param script: Mysqlsh script
+        :raises subprocess.CalledProcessError: Raises CalledProcessError if the
+                                               script gets a non-zero return
+                                               code.
+        :returns: subprocess output
+        """
```

Comment:
As Shayan commment, this can be changed to something like [this](https://github.com/canonical/mongodb-operator/blob/ac6c0ef8b2cb24f607b26cffa1ea0e94d5748692/src/mongod_helpers.py#L302) to have a pattern in the docstrings.